analysis/diabetes_analysis.ipynb

```diff
@@ -223,8 +226,337 @@
+       "      <td>112</td>\n",
+       "      <td>26.2</td>\n",
+       "      <td>0.245</td>\n",
+       "      <td>30</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>762</th>\n",
+       "      <td>1</td>\n",
+       "      <td>126</td>\n",
+       "      <td>60</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>30.1</td>\n",
+       "      <td>0.349</td>\n",
+       "      <td>47</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>763</th>\n",
+       "      <td>1</td>\n",
+       "      <td>93</td>\n",
+       "      <td>70</td>\n",
+       "      <td>31</td>\n",
+       "      <td>0</td>\n",
+       "      <td>30.4</td>\n",
+       "      <td>0.315</td>\n",
+       "      <td>23</td>\n",
+       "      <td>0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>764 rows × 9 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     Pregnancies  Glucose  BloodPressure  SkinThickness  Insulin   BMI  \\\n",
+       "0              6      148             72             35        0  33.6   \n",
+       "1              1       85             66             29        0  26.6   \n",
+       "2              8      183             64              0        0  23.3   \n",
+       "3              1       89             66             23       94  28.1   \n",
+       "4              0      137             40             35      168  43.1   \n",
+       "..           ...      ...            ...            ...      ...   ...   \n",
+       "759           10      101             76             48      180  32.9   \n",
+       "760            2      122             70             27        0  36.8   \n",
+       "761            5      121             72             23      112  26.2   \n",
+       "762            1      126             60              0        0  30.1   \n",
+       "763            1       93             70             31        0  30.4   \n",
+       "\n",
+       "     DiabetesPedigreeFunction  Age  Outcome  \n",
+       "0                       0.627   50        1  \n",
+       "1                       0.351   31        0  \n",
+       "2                       0.672   32        1  \n",
+       "3                       0.167   21        0  \n",
+       "4                       2.288   33        1  \n",
+       "..                        ...  ...      ...  \n",
+       "759                     0.171   63        0  \n",
+       "760                     0.340   27        0  \n",
+       "761                     0.245   30        0  \n",
+       "762                     0.349   47        1  \n",
+       "763                     0.315   23        0  \n",
+       "\n",
+       "[764 rows x 9 columns]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
```

Comment:
Seems this entire code chunk is a duplicate of `schema` commented out?

Suggestion: should be removed

Reply:
this was an error now removed